api/app/Http/Controllers/EntityController.php

```diff
@@ -174,21 +190,21 @@ public function patchOne($id, Request $request)
     public function patchMany(Request $request)
     {
         $requestCollection = $request->data;
-        $ids = $this->getIds($requestCollection, $this->getKeyName(), $this->validateRequestRule);
-        $models = $this->getRepository()->findMany($ids);
+        $ids = $this->getIds($requestCollection, $this->getModel()->getKeyName(), $this->validateRequestRule);
+        $models = $this->getManyEntitiesByIds($ids);
         if ($models->count() !== count($ids)) {
-            throw $this->notFoundManyException($ids, $models, $this->getKeyName());
+            throw $this->notFoundManyException($ids, $models, $this->getModel()->getKeyName());
```

Comment:
As mentioned above this could be extracted to a nice `$this->findOrFailCollection($requestCollection)` method.